charts/op-scim-bridge/values.yaml

```diff
@@ -204,11 +204,11 @@ redis:
               topologyKey: "kubernetes.io/hostname"
     resources:
       requests:
-        cpu: 125m
-        memory: 256M
-      limits:
         cpu: 250m
         memory: 512M
+      limits:
```

Comment:
So we're doubling the `limits` for the default values? This sounds like it should be ok, but I'm curious of the thought process.

Reply:
My mistake @chasdl. I thought we had upped all of the values. I've since reverted that change and it now matches the values you set in the scim-examples.

Reply:
I believe we are also doubling the default values for the SCIM pod above on [Line 105](https://github.com/1Password/op-scim-helm/pull/110/files#diff-e2d007e13e9f27b2aa5122c2a340fc4133c0107fb04b77104d4b6922982ff94dL105). 

The rest of this MR doesn't appear to match. Is that a missed revert or am I missing something?

Reply:
@grellyd That was just missed in the revert. I've since cleaned that up.